README.md

```diff
@@ -1,22 +1,50 @@
-# KCD Raid Group :crossed_swords:
+# Hi :wave:, Welcome to the OSS Raid Group :crossed_swords:
+
+The OSS Raid Group is what happens when you combine MMORPGs, Programming, and Open Source Software - a group dedicated to defeating the Raid Bosses of OSS
+
+For those who aren't aware of these 'gamer' terms, and even those that are; here are the definitions we use when operating as a Raid Group.
+
+| Phrase    | Definition |
+| --------- | ---------- |
+| Raid      | A Raid is the OSS project's issue that we are working to resolve - see how we [converted react-hooks-testing-library to TypeScript](https://github.com/testing-library/react-hooks-testing-library/pull/520) for an example |
+| Raid Boss | The Raid Boss is the issue that we are 'fighting' to resolve |
+
+## Storyline Quest (mission) :scroll:
 
-## Mission & What We Do :metal:
 - Themed from MMORPG gaming, gamifiying Mob programming OSS repos
 - Look for criteria in OSS repos 
   1. Low to medium contributor activity, only a handful of maintainers
   2. Medium to high usage of the OSS 
   3. Our impact can be potentially high; in a few days
 
-## Joining KCD Raid Group :smiling_face_with_three_hearts:
-1. Active Raid contributor
-2. Active Community member (KCD) 
-3. Determine responsibilities then set to either Org member or Collab (reduced permissions)
+## Join Us! :smiling_face_with_three_hearts::partying_face:
+
+Anoyone is welcome to participate in the raids; just raise an issue in the current raid repo, and [...TBD]
+
+### Intake Quests
+
+1. Say hello to your fellow raiders in the #🐝-raid-general channel on [Discord][discord]
+2. Raise your first issue against a raid, and start contributing!
 
-## The Process find target -> communicate -> Raid -> Push Code :fire:
-1. One Target at a time to maximize impact in short amount of time.
+### Advancements
+
+When you are:
+* Actively contributing to Raids
+* Being an active community member on the [KCD Discord][discord]
+
+Then we can determine responsibilities and get you running as either an Org member or Collab (reduced permissions)
+
+## The Process :fire:
+### `Find Target` :dart: `Communicate` :satellite: `Raid` :crossed_swords: `Push Code` :tada:
+
+One Raid at a time to maximize impact in short amount of time.
 
 ## Communication :smiley:
-1. Primarily www.kentcdodds.com/discord #oss-raid-group channel 
+
+We primarily communicate via [Discord][discord] in the `OSS-RAID-GROUP` category
+
+1. Primarily www.kentcdodds.com/discord OSS-RAID-GROUP Category, #:bee:-raid-general
```

Comment:
hm, the 4th link to discord might be bit too many..

Reply:
Yeah, agree with you there, we can just improve it later :tada:

Reply:
Yep - I just changed this to a draft (didn't realise you could make drafts!)

I will be making more changes later, but it's always good to get more eyes early on 😄